src/components/EventCardList/EventCardList.jsx

```diff
@@ -83,17 +85,19 @@ const EventCardList = ({eventList}) => {
 	return (
 		<EventListContainer data-testid="list">
 			{error && <div>An error occured: {error}</div>}
-			{data.map(singleEvent => {
-				return (
-					<EventCard
-						data-testid="listItem"
-						key={singleEvent._id}
-						event={singleEvent}
-						deleteEvent={deleteEvent}
-						bookmark={bookmarkEvent}
-					></EventCard>
-				);
-			})}
+			{data
```

Comment:
It is better to use another more specific name as data.

Reply:
Done